### PR TITLE
Summer Yasminkov 8000C gaiden + first scene of regular Yasminkov 8000C + some fixes

### PR DESCRIPTION
**Pull request by DabirSA**

### CODE REVIEW

File: json/Explain_Element_Booster.txt [hunk 1]
```diff
@@ -1607,7 +1607,7 @@
 		"jp_text": "エクストレセプター",
 		"tr_text": "Extreceptor",
 		"jp_explain": "アルター、フリクト系と\n特定の特殊能力継承率を\n大幅に上昇　全+10",
-		"tr_explain": "Boosts Flict/Alter-type\nability transfer rates.\nAll +10"
+		"tr_explain": "Boosts Flict, Alter and\ncertain other special\nability transfer rates.\nAll +10"
```
Comment (DabirSA) [inline json/Explain_Element_Booster.txt]: Ext also boosts Stat 4 and 5, I think that's what the description's alluding to.

File: json/Side_Text.txt [hunk 2]
```diff
@@ -177596,191 +177596,191 @@
...
+		"tr_text": "It's alright, really. My name is Gene.<%br>This is More, and this is Leader!",
 		"fileID": 170
 	},
 	{
 		"eventNo": "12",
 		"jp_name": "ヤスミノコフ８０００Ｃ",
 		"tr_name": "Yasminkov 8000C",
 		"jp_text": "こ、この通り、未熟な半人前であります！<%br>ご指導、よろしくお願いするでありますっ！",
-		"tr_text": "",
+		"tr_text": "I'm so grateful that you agreed to work with someone<%br>as low as me! I hope to learn a lot from you!",
```
Comment (SynthSy) [inline json/Side_Text.txt]: > こ、この通り、未熟な半人前であります！
> I'm so grateful that you agreed to work with someone as low as me!
This isn't rolling off the tongue right. It should be something like inexperienced.

File: json/Side_Text.txt [hunk 3]
```diff
@@ -177596,191 +177596,191 @@
...
+		"tr_text": "It's very nice to meet you, too.<%br>I'm sure we'll learn a lot from each other.",
 		"fileID": 170
 	},
 	{
 		"eventNo": "14",
 		"jp_name": "セラフィ",
 		"tr_name": "Seraphy",
 		"jp_text": "８０００Ｃさんの育成を兼ねた複数の<%br>任務には、それぞれサポートが入ります。",
-		"tr_text": "",
+		"tr_text": "Of course, when you're training 8000C,<%br>you'll have a support team to back you up.",
```
Comment (SynthSy) [inline json/Side_Text.txt]: This line says `While you're training 8000C, a support team will be there to assist you in your mission.`

File: json/Side_Text.txt [hunk 4]
```diff
@@ -177596,191 +177596,191 @@
...
+		"tr_text": "O-okay! Whatever you say, 2000H!<%br>It's an honor to be serving with you!",
 		"fileID": 170
 	},
 	{
 		"eventNo": "21",
 		"jp_name": "ヤスミノコフ８０００Ｃ",
 		"tr_name": "Yasminkov 8000C",
 		"jp_text": "（は、はわぁ……２０００Ｈ先輩だぁ……<%br>　正確無比、任務達成率１００％の！）",
-		"tr_text": "",
+		"tr_text": "(W-wow... It's really 2000H! I've heard he has<%br> perfect accuracy, and he's never failed a mission!)",
```
Comment (SynthSy) [inline json/Side_Text.txt]: remove the space after the `<%br>`.

**Pull request merged**